layouts/_default/landing.html

```diff
@@ -0,0 +1,192 @@
+        <p class="mt-4">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+      </div>
+
+    </div>
+  </section>
+  <section class="bg-s-2 py-14 px-2">
+    <div class="bg-white mx-auto flex justify-evenly my-14 px-4 pt-10 pb-10 font-serif max-w-screen-lg border-2 border-red rounded">
+      <h2 
+        class="font-sans text-2xl sm:text-3xl md:text-4xl font-black leading-none tracking-wide flex-[0_1_628px]"
+      >
+        Join the Leaders in Action
+      </h2>
+      <a href="#donate-leader"
+        class="flex items-center justify-center font-sans outline-none inline-block rounded px-3 py-2 text-base font-semibold tracking-wide ring ring-transparent bg-red-6 text-white hover:bg-red-5 focus:ring-red-7 active:ring-red-9"
+      >
+        Contribute &rarr;
+      </a>
+    </div>
+
+    <div class="mx-auto w-full max-w-screen-md my-14">
+      <h2 
+        class="font-sans text-2xl sm:text-4xl md:text-5xl font-black leading-none tracking-wide"
+      >
+        Current Leaders
+      </h2>
+
+      <div class="supporters-list">
+        <ul
+          class=" columns-2 md:columns-3 column-gap-2 py-8" 
+          style="column-gap:40px;"
+        >
+          {{ range sort site.Data.leaders.list "sort" "asc" }}
+            <li>{{ .display }}</li>
+          {{ end }}
+        </ul>
+      </div>
+
+    </div>
+  </section>
+
+  <div id="donate-leader" class="px-2 md:px-20 py-14 my-0 bg-blue">
+    {{ partial "tw/donate-prefill-lg.html" (dict
+      "page" .
+      "hed" "Support Nonpartisan, Independent Journalism"
+      "cta" "Contribute Now"
+      )
+    }}
+  </div>
+
+  </div>
+{{ end }}
+
+{{ define "extra-main" }}
+  {{ $pages := $.Pages }}
+  {{ if .Param "sort-by" | eq "title" }}
+    {{ $pages = $.Pages.ByTitle }}
+  {{ end }}
+
+
+{{ end }}
+
+{{ define "before-footer" }}
+  {{ partialCached "tw/promo-footer.html" (dict
```

Comment:
We might want to drop this ad, since the page ends with an ad for the donate prefill box anyway.